Simplenote/src/main/java/com/automattic/simplenote/PreferencesFragment.java

```diff
@@ -475,17 +476,36 @@ public int compare(String text1, String text2) {
 
     private void importData(Uri uri) {
         try {
-            Importer.fromUri(this, uri);
+            AppLog.add(Type.IMPORT, "Importing notes from " + uri + ".");
+
+            FragmentActivity activity = getActivity();
+            if (activity == null) {
+                AppLog.add(Type.IMPORT, "Could not import notes since activity is null");
+                return;
+            }
+
+            Importer.fromUri(activity, uri);
             toast(R.string.import_message_success);
+
+            AppLog.add(Type.IMPORT, "Notes imported correctly!");
         } catch (Importer.ImportException e) {
             switch (e.getReason()) {
                 case FileError:
+                    AppLog.add(Type.IMPORT, "File error while importing note. Exception: "
```

Comment:
Minor (🔍): Consider making the `AppLog` a one-liner since it actually fits.

PS: The same applies for those other two `AppLog` lines further below.

Reply:
@ParaskP7 I am using 100 characters line length (default) and if I put the message, it overflows the 100 characters. Do you have another value (maybe 120)?

Reply:
Oh yea, `120` is the default in general and we are using that on `WPAndroid` and `WCAndroid` as far as I am aware. I would recommend maybe having `120` on Simplenote for consistency purposes as well. 😊

Reply:
Sounds good, thanks! Fixed in de71d5b472ca3060559429ddeb42176241e00101.